tests/repl.sh

```diff
@@ -71,3 +73,48 @@ testReplResponse '
 :a { a = "2"; }
 "result: ${a}"
 ' "result: 2"
+
+testReplResponse '
+drvPath
+' '".*-simple.drv"' \
+$testDir/simple.nix --experimental-features ''
+
+testReplResponse '
+drvPath
+' '".*-simple.drv"' \
+--file $testDir/simple.nix --experimental-features ''
+
+testReplResponse '
+drvPath
+' '".*-simple.drv"' \
+--file $testDir/simple.nix --experimental-features 'repl-flake'
```

Comment:
```suggestion
--file $testDir/simple.nix --extra-experimental-features 'repl-flake'
```

To make it work with the CA-drvs test

And same below